spec/bolt/module_installer/resolver_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'puppetfile-resolver'
+require 'bolt/module_installer/resolver'
+
+describe Bolt::ModuleInstaller::Resolver do
+  let(:name)      { 'yaml' }
+  let(:full_name) { "puppetlabs/#{name}" }
+  let(:version)   { '0.1.0' }
+  let(:git)       { "https://github.com/puppetlabs/puppetlabs-#{name}" }
+  let(:ref)       { '0.1.0' }
+  let(:mod)       { forge_module }
+  let(:resolver)  { described_class.new }
+  let(:spec)      { double('spec', to_resolver_module: mod, name: name, git: git, sha: ref) }
+  let(:specs)     { double('specs', specs: [spec]) }
+
+  let(:forge_module) do
```

Comment:
This doesn't seem to get used?

Reply:
Line 13 sets `forge_module` as the value for `mod`, and line 15 sets `mod` as the return value for the `to_resolver_module` stub.